README.md

```diff
@@ -84,3 +104,8 @@ If you need to add some other kind of static file, it should go somewhere in the
 
 ### Maki Icons
 https://labs.mapbox.com/maki-icons/
```

Comment:
```suggestion
```

This can be removed. I added this as a note to self on the first night of development when trying to do the markers in a Mapbox symbol layer. It has no use in the README now.

Reply:
Thanks for the heads up. I removed this in the most recent commit.